pkg/systemd/services/service-details.jsx

```diff
@@ -187,7 +187,6 @@ const ServiceActions = ({ masked, active, failed, canReload, actionCallback, del
                   isOpen={isActionOpen}
                   isPlain
                   onSelect={() => setIsActionOpen(!isActionOpen)}
-                  position='right'
                   dropdownItems={actions} />
```

Comment:
```suggestion
                  className='pf-u-flex-grow-on-sm pf-u-justify-content-flex-end-on-sm'
                  alignments={{ default: 'right', sm: 'left' }}
                  dropdownItems={actions} />
```

Reply:
It should probably be something like this for a proper fix. Screenshots in the comment I posted on the issue.